actions/feedback-service.js

```diff
@@ -99,24 +99,31 @@ function update(feedbackClass, cloudant, dbName, minrpi, maxrpi, activities) {
           if (er) {
             reject(er);
           }
-          results.min_rpi = 300;
-          results.max_rpi = 0;
+          results.min_rpi = Infinity;
+          results.max_rpi = -Infinity;
           results.docs.forEach(result => {
-            if (result.rpi < results.min_rpi) {
-              results.min_rpi = result.rpi;
+            if (result.signDiff < results.min_rpi) {
+              results.min_rpi = result.signDiff;
             }
-            if (result.rpi > results.max_rpi) {
-              results.max_rpi = result.rpi;
+            if (result.signDiff > results.max_rpi) {
+              results.max_rpi = result.signDiff;
             }
           });
+
+          results.sort((a, b) => {
+            return a.absDiff - b.absDiff;
+          });
+          results = results.slice(0, 5);
```

Comment:
We only return the top 5 results, but return the max and min from all the results. The next call will have no results despite the fact there are results in the db (assuming we have more than 5 more results.